packages/oauth-twitter/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as oauth from 'oauth';
```

Comment:
Just to confirm: if we're using `oauth` package does this mean we can remove the `grant` dependency overall?

Reply:
Not really the tokens passed down to the oauth package came from grant. We need to verify these token return the good user, this is why we have to use the oauth module here.